components/tekton/tasks/set-tag.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: set-tag
+spec:
```

Comment:
Could you please add a description for this task?

Reply:
Done.